### PR TITLE
Enable SA1136: Enum values should be on separate lines #94098

### LINKED ISSUE #94098: SA1136: Enum values should be on separate lines

### PR DESCRIPTION
**Pull request by artl93**
Fixes #94098, enabling another style rule.

Includes a couple places where the rule does not make sense, but otherwise, this includes all the code clean-up as well. 

Style updates were done by hand.

### CODE REVIEW

File: src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsWrite.cs [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 namespace BasicEventSourceTests
 {
+#pragma warning disable SA1136 // Enum values should be on separate lines
```
Comment (danmoseley) [inline src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsWrite.cs:16]: Or you could enable the rule for product only not tests. We do that with some rules.

**Review by danmoseley — APPROVED** [review]
Source changes look good, I defer to others on whether it would be info or warning and whether should include tests

**Comment by danmoseley** [comment]
Btw it's good to mention whether cleanup was done by hand or with a fixer, as the latter hopefully needs less careful eyeballing...

**Review by danmoseley — APPROVED** [review]

**Review by stephentoub — APPROVED** [review]
LGTM

**Comment by danmoseley** [comment]
BTW, we often put a comma on the last enum entry, to reduce diffs when adding to the enum. I didn't suggest that because it didn't seem that important. But maybe if there was a fixer for this, it should do that.

**Pull request merged**